frontend/pages/admin/IntegrationsPage/cards/AutomaticEnrollment/components/AppleBusinessManagerSection/AppleBusinessManagerSection.tsx

```diff
@@ -113,6 +134,7 @@ const AppleBusinessManagerSection = () => {
       return <DataError />;
     }
 
+    // no error, but no apple bm data yet. TODO: when does this happen?
```

Comment:
That's the 404 case, no?

Reply:
yep exectly. just a differnt way of writing it. I'll come back and change a bit later as I'm unsure why we did it that way originally so am hesitant to change in this PR